InternTrack.Core.Api.Infrastructure.Build/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using ADotNet.Clients;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.SetupDotNetTaskV1s;
+
+var githubPipeline = new GithubPipeline
+{
+    Name = "InternTrackCoreApi",
+
+    OnEvents = new Events
+    {
+        PullRequest = new PullRequestEvent
+        {
+            Branches = new string[] { "main" }
+        },
+
+        Push = new PushEvent
+        {
+            Branches = new string[] { "main" }
+        }
+    },
+
+    Jobs = new Jobs
+    {
+        Build = new BuildJob
+        {
+            RunsOn = BuildMachines.Windows2022,
+
+            Steps = new List<GithubTask>
+            {
+                new CheckoutTaskV2
+                {
+                   Name = "Checking Out Code"
+                },
+                 new SetupDotNetTaskV1
+            {
+                Name = "Installing .NET",
+
+                TargetDotNetVersion = new TargetDotNetVersion
+                {
+                    DotNetVersion = "7.0.203",
+                    IncludePrerelease = true,
+                }
+
+               },
+                 new RestoreTask
+                 {
+                     Name = "Restoring Nuget Packages"
+                 },
+
+                 new DotNetBuildTask
+                 {
+                     Name = "Building Project"
```

Comment:
indentations here aren't done right. This seems like a copied code. Do it line by line by hand.